build.gradle

```diff
@@ -49,7 +49,10 @@ configurations {
 }
 
 dependencies {
-	compile("net.sengir.forestry:forestry_1.7.10:${config.forestry.version}:dev") {
+	compile("net.sengir.forestry:forestry_1.7.10:${config.forestry.version}:api") {
+		transitive = false
+	}
+	runtime("net.sengir.forestry:forestry_1.7.10:${config.forestry.version}:dev") {
 		transitive = false
```

Comment:
Just makes it so it can't accidentally build against internal stuff, only the API is available at compile time.
The dev jar is still there when it's being run.